.readthedocs.yaml

```diff
@@ -19,5 +25,8 @@ sphinx:
 # We recommend specifying your dependencies to enable reproducible builds:
 # https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
 # python:
-#   install:
-#   - requirements: docs/requirements.txt
\ No newline at end of file
+#  install:
+#  - method: pip
+#    path: .
+#    extra_requirements:
+#    - foo
```

Comment:
Is "foo" a copy&paste left-over from some example config?

Reply:
Foo is just a placeholder. Everything in the comments is just a reminder.